src/int_vectors/compact_vector.rs

```diff
@@ -25,13 +24,13 @@ use crate::{utils, Serializable};
 /// // Can store integers within 3 bits each.
 /// let mut cv = CompactVector::new(3)?;
 ///
-/// cv.push_int(7)?;
-/// cv.push_int(2)?;
+/// cv.push_int(7);
+/// cv.push_int(2);
```

Comment:
```suggestion
/// cv.push_int(7)?;
/// cv.push_int(2)?;
```
The `?` operators should not be omitted. The example code in Rustdoc also works as unit tests, so it needs to correctly find any errors.

Reply:
Sorry, I mixed it up with other functions that don't require the `?` operator anymore, I will accept the revert on this change.